code-for-nashville/nashviva
# Media Query: Nav Opacity

Issue #62 
**Alternate solution:**
Set a lower opacity for the background color on `nav` to allow the user a less inhibited view of the map.

_To Test:_
Please set viewport to <450px wide to see change in opacity
or use device toolbar in dev tools.
